notebooks/SITCOMTN_131_M1M3_VMS_TMA_Coupling_investigation_strong_shaking-28-06-2023.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.            print(t_peak)
</pre><p>Remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-131/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I think that it is interesting to have the peak positions printed out in order to find which ones are common to several sensors / axis. </p><p>I improved the format to make it nicer and clearer</p>